helm/flowforge/templates/deployment.yaml

```
@@ -8,7 +8,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.forge.replicas | default 1 }}
```

Since we setup default value in the chart's `values.yaml` this one is redundant.
```suggestion
  replicas: {{ .Values.forge.replicas }}
```